mmv1/templates/terraform/examples/pubsub_subscription_different_project.tf.erb

```diff
@@ -6,5 +6,5 @@ resource "google_pubsub_topic" "<%= ctx[:primary_resource_id] %>" {
 resource "google_pubsub_subscription" "<%= ctx[:primary_resource_id] %>" {
   project = "<%= ctx[:vars]['subscription_project'] %>"
   name    = "<%= ctx[:vars]['subscription_name'] %>"
-  topic   = google_pubsub_topic.<%= ctx[:primary_resource_id] %>.name
+  topic   = google_pubsub_topic.<%= ctx[:primary_resource_id] %>.id
```

Comment:
To make sure this works as expected, could you extend the example to include setting up a second that's being accessed and remove [skip_test](https://github.com/GoogleCloudPlatform/magic-modules/blob/f8feaf07fdeff43bd80832c47ec15f645d7228a4/mmv1/products/pubsub/Subscription.yaml#L48)?

Reply:
Sorry, I'm not sure what you mean by "a second" here.

Reply:
wow, yeah, that sentence is incomprehensible - sorry about that! I meant a second project. If we were running this as a test, we'd want to make sure that we were actually setting up a second _project_ (in addition to the default test project) and that the topic was in the other project. Currently the example uses `topic-project` but doesn't actually _create_ that project.

But as you pointed out, if the rest of the pubsub documentation uses `.id` consistently, then cross-project doesn't need to be documented at all since it won't work differently than the other cases.